### PR TITLE
feat(config): add links to device manuals

### LINKED ISSUE #1551: [feat] Add links to z-wave manual or zwave-js configs file
**Is your feature request related to a problem? Please describe.**
Sometimes when working on devices, it's useful to read more about configs or behaviour in the manuals available. In z-wave, this is doubly easy, as the z-wave standardization seems to mean that manuals are available at a central location.

**Describe the solution you'd like**
Alongside the export button/link for the node json, you could add a link to the relevant zwave-js config file, and the manual on z-wave.info (or other place .. I'm not entirely sure how to find them).

### Config file:

For the config file, you can link to something like this. In some cases, the config file has a version range on it (some of the files in this directory have that). Maybe zwave-js can tell you which config file is being used? - alternatively just linking to the model number for that manufacturer is a good start (remember to not link when manufacturer is 0x0000 :)).

> https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x019a/11-01-011.json

### Manuals
Sometimes when searching for model numbers on google, I find these links:

> https://manuals.zwave.eu/backend/make.php?lang=EN&sku=ZWA012-C&cert=ZC12-20060075&type=mini
> http://manuals-backend.z-wave.info/make.php?lang=en&sku=ZW112-B&cert=ZC10-16010007
> http://manuals-backend.z-wave.info/make.php?lang=en&sku=ZWA008-C&cert=ZC10-19066568

If possible, it'd be cool to link to these, or better manuals if available.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by blhoward2**
Provides the info to integrate a link to the manual, which will close #1551. @marcus-j-davies @robertsLando

### CODE REVIEW

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@AlCalzone These should be ready to merge. The only change is adding the manual link, and anything else that normalizeConfig() caught.

**Comment by robertsLando** [comment]
Nice!! :)

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 1]
```diff
@@ -236,7 +123,7 @@
 			"label": "Key Pressed 1 time – value of SWITCH ON command sent to 2nd association group",
 			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: Uh I think these are a mistake

File: packages/config/config/devices/0x0131/ne_nas_ab02z.json [hunk 2]
```diff
@@ -85,9 +85,9 @@
 		"3": {
 			"label": "Configuring doorbell music duration time",
 			"description": "Duration",
-			"unit": "seconds",
 			"valueSize": 1,
-			"minValue": 0,
+			"unit": "seconds",
+			"minValue": 255,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne_nas_ab02z.json]: Also this

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Please check the other changes that they don't introduce errors like the two I found. The other diffs are not expanded on GitHub and I currently don't have time to open every one by hand.

**Comment by blhoward2** [comment]
I'm just going to revert and re-run without the normalizeConfig for now. I don't have time to go through them all either. And come to think of it, we've already touched all of these once already with that.

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@AlCalzone If you can invoke the bot to fix the lint, these are ready to merge. I did a complete revert and removed the normalizeConfig, so we're only adding the manual link and then writing back.

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Review by blhoward2 — COMMENTED** [review]
@AlCalzone Ready to go

File: packages/config/config/devices/0x0131/ne_nas_ab02z.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
 			"label": "Configure alarm volume",
 			"description": "Low, Medium, High",
 			"valueSize": 1,
-			"minValue": 1,
+			"minValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne_nas_ab02z.json]: We don't have an option for 0

File: packages/config/config/devices/0x0131/ne_nas_ab02z.json [hunk 4]
```diff
@@ -60,7 +60,7 @@
 			"label": "Configure alarm duration time",
 			"description": "30 sec, 1 min, 5 min",
 			"valueSize": 1,
-			"minValue": 1,
+			"minValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne_nas_ab02z.json]: dito

**Review by AlCalzone — DISMISSED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**